### ISSUE TITLE
Vizy field is hidden / disappeared when changing entry type - timeout too quick?

### ISSUE THREAD

**Issue by bleepsandblops** [verbb/vizy#167]
### Describe the bug

If I create a new entry for which the default entry type does not have a Vizy field, then switch the entry type to one that does have a Vizy field, the Vizy field is hidden:
<img width="100" alt="Screenshot 2022-08-06 at 22 42 21" src="https://user-images.githubusercontent.com/1942313/183266939-93470977-2283-4bdb-bc71-c7c6231f2bc9.png">
and in the console I see this:
<img width="442" alt="Screenshot 2022-08-06 at 22 42 44" src="https://user-images.githubusercontent.com/1942313/183266947-c86640a4-6872-4b30-b5ac-ae70cddcd3af.png">
which refers to this line:
`setTimeout(function() { new Craft.Vizy.Input("fields-contentVizy", "fields[contentVizy]"); }, 200);
`

If I then manually run that line in the console it works and the field appears! So this makes me think this timeout is a bit dodgy and is too short.

### Steps to reproduce

1. Have a section with two entry types, the default one being one without a Vizy field, the other one with one
2. Create a new entry, then swap the entry type to the type that has a Vizy field
3. The field does not appear

### Craft CMS version

4.2.0.2

### Plugin version

2.0.4

### Multi-site?

No

### Additional context

If I open the entry after, the field is displaying correctly.

**Comment by engram-design** [comment]
Yep, `setTimeout` was always a dodgy stop-gap. Fixed for the next release. To get this fix early, change your `verbb/vizy` requirement in `composer.json` to:

```
"require": {
  "verbb/vizy": "dev-craft-4 as 2.0.4",
  "...": "..."
}
```

Then run `composer update`.

**Comment by bleepsandblops** [comment]
Thanks for your quick reaction @engram-design that seems fine now!

**Comment by engram-design** [comment]
Fixed in [2.0.5](https://github.com/verbb/vizy/releases/tag/2.0.5)
